ui/src/app/applications/components/application-status-panel/application-status-panel.tsx

```diff
@@ -30,12 +30,13 @@ const sectionLabel = (info: SectionInfo) => (
     </label>
 );
 
-const sectionHeader = (info: SectionInfo, onClick?: () => any) => {
+const sectionHeader = (info: SectionInfo, hasMultipleSources: boolean, onClick?: () => any) => {
     return (
         <div style={{display: 'flex', alignItems: 'center'}}>
             {sectionLabel(info)}
             {onClick && (
-                <button className='argo-button argo-button--base-o argo-button--sm application-status-panel__more-button' onClick={onClick}>
+                <button className='argo-button argo-button--base-o argo-button--sm application-status-panel__more-button' onClick={onClick} disabled={hasMultipleSources}>
+                    {hasMultipleSources && (helpTip("More button is not supported for apps with multiple sources"))} 
```

Comment:
```suggestion
                    {hasMultipleSources && (helpTip("Sync status details are not currently available for apps with multiple sources."))} 
```

Reply:
The More button is same for `Sync Status` and `Sync Result`. Thus, kept it same as "More button" for it to work in both cases.

Also `currently available` states that the details are expected to become available in some time or so, which is not the case.

I think below change might be better.. Let me know wdyt..

```
More details are not currently supported for apps with multiple sources.
```